src/app/features/tech-record/components/tech-record-summary/tech-record-summary.component.ts

```diff
@@ -122,13 +124,13 @@ export class TechRecordSummaryComponent implements OnInit, OnDestroy {
   get customSectionForms(): Array<CustomFormGroup | CustomFormArray> {
     const commonCustomSections = [this.body?.form, this.dimensions?.form, this.tyres?.form, this.weights?.form];
 
-    switch (this.techRecordCalculated.vehicleType) {
+    switch (this.techRecord.vehicleType) {
```

Comment:
This breaks the component. It needs reverting.

Why was it changed?

Reply:
No idea when that was changed